extensions/emmet/src/abbreviationActions.ts

```diff
@@ -445,9 +444,7 @@ export function isValidLocationForEmmetAbbreviation(document: vscode.TextDocumen
 
 	if (currentHtmlNode) {
 		if (currentHtmlNode.name === 'script') {
-			return (currentHtmlNode.attributes
-				&& currentHtmlNode.attributes.some(x => x.name.toString() === 'type'
-					&& allowedMimeTypesInScriptTag.indexOf(x.value.toString()) > -1));
+			return isTemplateScript(currentHtmlNode);
```

Comment:
The check on the `name` is already done inside the `isTemplateScript` function. So we can do a simple 
```
if (isTemplateScript(currentHtmlNode)) {return true;}
```